src/test/appdistro/client.spec.ts

```diff
@@ -39,17 +41,44 @@ describe("distribution", () => {
 
   describe("getApp", () => {
     it("should throw error when app does not exist", () => {
-      nock(api.appDistributionOrigin).get(`/v1alpha/apps/${appId}`).reply(404, {});
+      nock(api.appDistributionOrigin)
+        .get(`/v1alpha/apps/${appId}?appView=${appViewBasic}`)
+        .reply(404, {});
       return expect(appDistributionClient.getApp()).to.be.rejected;
     });
 
     it("should resolve when request succeeds", () => {
-      nock(api.appDistributionOrigin).get(`/v1alpha/apps/${appId}`).reply(200, {});
+      nock(api.appDistributionOrigin)
+        .get(`/v1alpha/apps/${appId}?appView=${appViewBasic}`)
+        .reply(200, {});
       return expect(appDistributionClient.getApp()).to.be.fulfilled;
```

Comment:
What are these supposed to be fulfilled with? The returned value isn't being checked, which seems like an oversight.

Reply:
good question, this was just an update of an existing test. I'll take a look at this.

Reply:
@bkendall I may just call appDistributionClient.getApp() without an expectation. We're mocking the response, so ultimately the response will be whatever we mock. We're really testing this to ensure that we send the correct request to the backend, and less to verify the response. What do you think?